index.php

```diff
@@ -59,10 +57,18 @@ function searchedValue(): string {
 
     <div class="col">
         <div class="row justify-content-center">
-            <form class="d-flex flex-row" action="">
-                <input name="search" type="search" class="form-control m-2" id="search" placeholder="<?php echo $searchedValue; ?>">
+            <form class="d-flex flex-row align-items-center" action="">
+                <input name="search" type="search" class="form-control m-2" id="search" value="<?php echo $searchedValue; ?>">
+                <select name="filter" class="btn m-1">
+                    <option value="">Filter by Food Type</option>
+                    <option value="Herbivore">Herbivore</option>
+                    <option value="Omnivore">Omnivore</option>
+                    <option value="Carnivore">Carnivore</option>
+                </select>
```

Comment:
Could this also display the currently selected filter?

Reply:
OOO that would be nice